.github/workflows/test.yml

```diff
@@ -776,6 +776,92 @@ jobs:
         with:
           commit_message: Benchmark result for ${{ steps.date.outputs.pretty }} (${{ github.sha }})
 
+  next_js_integration:
+    needs: [determine_jobs, rust_prepare, rust_build_release]
+    if: needs.determine_jobs.outputs.rust == 'true' && (github.event_name == 'pull_request' || needs.determine_jobs.outputs.push == 'true')
+    name: Next.js integration test
+    runs-on: ubuntu-latest-8-core-oss
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [16, 18]
```

Comment:
Testing this on two node.js version seems like an overkill to me?

Reply:
This is copy-paste from what next.js does. Not 100% sure if we'll need or not, but wanted to start from 1:1 parity.

Reply:
To clarify, for now I'd like to keep this for a while.

Reply:
ok